### PR TITLE
Various typos (x86 paging)

### PR DESCRIPTION
**Pull request by acharron**
Correction of some minor typos in the x86 article.

### CODE REVIEW

File: x86-paging.md [hunk 1]
```diff
@@ -491,7 +491,7 @@ and after a second translation of `00007` to `00009` it becomes:
 
 Now if `00003` needs to be translated again, hardware first looks up the TLB and finds out its address with a single RAM access `00003 --> 00005`.
 
-Of course, `00000` is not on the TLB since on valid entries contain `00000` as key.
+Of course, `00000` is not on the TLB since no valid entry contain `00000` as a key.
```
Comment (cirosantilli) [inline x86-paging.md]: contains , but I'll fix on another commit :-)

**Comment by cirosantilli** [comment]
Thank you very much!! If you find any other errors let me know!

File: x86-paging.md (hunk 1 above)
Comment (acharron) [inline x86-paging.md]: Oh that's true, something I overlooked, my bad.

**Comment by acharron** [comment]
You're welcome. Your page was very useful to me, and I'm glad I could 
correct some minor mistakes easily.

I'm still learning how to manipulate GitHub more efficiently, so excuse 
me for the 2 pull requests that could have been made into only one.

**Comment by cirosantilli** [comment]
No worries ;)

BTW, the usual convention is to keep 1 commit per PR. But it doesn't matter so much here since it is prose.

You may also want to have a look into `git rebase` and force pushing.

**Pull request merged**